apps/src/code-studio/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -177,6 +177,14 @@ export default class WorkshopTable extends React.Component {
       }
     });
 
+    columns.push({
+      property: 'regional_partner_name',
+      header: {
+        label: 'Regional Partner',
+        transforms: [sortable]
+      }
+    });
```

Comment:
Any reason not to combine with the above `columns.push` call? Note, [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push) can take multiple arguments.

Reply:
oops, no reason :)